examples/samd/simplesample_http/simplesample_http.ino

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Arduino. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#include <SPI.h>
+
+#ifdef ARDUINO_SAMD_FEATHER_M0
+#include <Adafruit_WINC1500.h>
+#include <Adafruit_WINC1500Client.h>
+#include <Adafruit_WINC1500MDNS.h>
+#include <Adafruit_WINC1500Server.h>
+#include <Adafruit_WINC1500SSLClient.h>
+#include <Adafruit_WINC1500Udp.h>
+#elif defined(ARDUINO_SAMD_ZERO) || defined(ARDUINO_SAMD_MKR1000)
+#include <WiFi101.h>
+#endif
+
```

Comment:
Since we decided to split the ESP8226 example, I think it would be good to also split the Arduino board examples from the Feather boards examples. (Avoid the ifdef's all together).